cmd/eksctl-anywhere/cmd/installpackages.go

```diff
@@ -66,6 +61,11 @@ var installPackageCommand = &cobra.Command{
 }
 
 func runInstallPackages(cmd *cobra.Command, args []string) error {
+	if len(ipo.clusterName) > 0 {
+		ipo.source = curatedpackages.Cluster
```

Comment:
Can we update this here instead: https://github.com/aws/eks-anywhere/blob/21e9c7d54ccea75045145d8223a398614c57d22b/pkg/curatedpackages/bundle.go#L50.